urbansearch/clustering/preprocessing.py

```diff
@@ -0,0 +1,40 @@
+#Tokenizes, Stems, removes stop words and counts
+
+import re
+
+from nltk.tokenize import word_tokenize
+from nltk.stem.snowball import SnowballStemmer
+from nltk.corpus import stopwords
+from collections import Counter
+
+class PreProcessor:
+    def __init__(self, lang='dutch'):
+        self.lang = lang
+        self.stemmer = SnowballStemmer(self.lang)
+
+    def preProcess(self, text):
```

Comment:
Python convention is to never use capitals in function/method names, rather use an underscore: `pre_process`